gql/resolver_maddr.go

```diff
@@ -3,3 +3,11 @@ package gql
 func (r *resolver) MinerAddress() string {
 	return r.provider.Address.String()
 }
+
+func (r *resolver) GraphsyncRetrievalMinerAddresses() []string {
+	addrs := r.cfg.Dealmaking.GraphsyncStorageAccessApiInfo
```

Comment:
Isn't the `GraphsyncStorageAccessApiInfo` in the form of `xxx:/ip4/127.0.0.1/tcp/1234/http` and not a miner address?

Reply:
Good catch 👍 
I moved the config param so my manual test wasn't doing what I thought it was.